app/src/main/java/eu/kanade/tachiyomi/data/backup/full/models/Backup.kt

```diff
@@ -0,0 +1,18 @@
+package eu.kanade.tachiyomi.data.backup.full.models
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.protobuf.ProtoNumber
+
+/**
+ * Backup json model
+ */
+@ExperimentalSerializationApi
+@Serializable
+data class Backup(
```

Comment:
We may want to introduce a version field too.

Reply:
If we need it later on, we can add it and have it default to version 1, and have updated backups export as version 2